[ENH] add warning message to illogical parameter combination passed to `signal.clean` (#3003)
nilearn/nilearn: doc/whats_new.rst

  When `two_sided` is `False`, only values greater than or equal to the threshold
  are retained.

> - :func:`nilearn.signal.clean` raises warning when the user pass `False` to 

Reviewer: ```suggestion
- :func:`nilearn.signal.clean` raises a warning when the user sets 
```

>   parameter `detrend` and `standardize_confound`. The user is suggested to either

Reviewer: ```suggestion
  parameters `detrend` and `standardize_confound` to False.
  The user is suggested to set one of
```

>   those options as `True`, or standardize/demean the confounds before using the 

Reviewer: ```suggestion
  those options to `True`, or standardize/demean the confounds before using the 
```
